receptai.api/Controllers/SubfoodditController.cs

```diff
@@ -0,0 +1,99 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using receptai.api.Dtos.Subfooddit;
+using receptai.api.Helpers;
+using receptai.api.Interfaces;
+using receptai.api.Mappers;
+using receptai.api.Repositories;
+using receptai.data;
+
+namespace receptai.api.Controllers;
+
+[Route("api/subfooddit")]
+[ApiController]
+public class SubfoodditController : ControllerBase
+{
+    private readonly ISubfoodditRepository _subfoodditRepository;
+
+    public SubfoodditController(ISubfoodditRepository subfoodditRepository)
+    {
+        _subfoodditRepository = subfoodditRepository;
+    }
+
+    [HttpGet]
+    public async Task<IActionResult> GetAll(
+    [FromQuery] QuerySubfooddit query)
+    {
+        var subfooddits = await _subfoodditRepository.GetAllAsync(query);
+        var subfoodditDto = subfooddits.Select(r => r.ToSubfoodditDto());
```

Comment:
Should use "s" instead of "r". "r" stands for "recipe", does not make much sense here